x/clock/keeper/clock.go

```diff
@@ -0,0 +1,167 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/CosmosContracts/juno/v18/x/clock/types"
+)
+
+// Store Keys for clock contracts (both jailed and unjailed)
+var (
+	StoreKeyContracts       = []byte("contracts")
+	StoreKeyJailedContracts = []byte("jailed-contracts")
```

Comment:
this worries me vs a single Contracts array that stores a bool for isJailed within it. Logically a lot of true & false checks for double querying data concerns me in relation to logic issues down the line as the module expands

StoreKeyContracts = []byte("contracts")

where each contract is

```
contract {
    contract: address (string or bytes)
    isJailed: bool
}
```

Reply:
Restored to original design!